civis/io/_files.py

```diff
@@ -160,14 +159,15 @@ def _upload_part(item):
 
     # upload each part
     try:
-        with tempfile.TemporaryDirectory() as tmp_dir:
+        with TemporaryDirectory() as tmp_dir:
+            tmp_path = os.path.join(tmp_dir, 'file_to_civis')
```

Comment:
is it possible multiple calls might be made simultaneously?  Should we use a uuid instead of "file_to_civis" as the string here?

Reply:
Or will `TemporaryDirectory` guaranteed to be a unique folder name?